docs/installation.rst

```diff
@@ -0,0 +1,103 @@
+Installation
+============
+
+The page has information for installing ``openfe``, installing software
+packages that integrate with ``openfe``, and testing that your ``openfe``
+installation is working.
+
+``openfe`` currently only works on POSIX system (macOS and UNIX/Linux). It
+is tested against Python 3.9 and 3.10.
+
+Installing ``openfe``
+---------------------
+
+When you install ``openfe`` through any of the methods described below, you
+will install both the core library and the command line interface.
+
+Installation with ``conda`` (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We recommend installing ``openfe`` with ``conda``, because it provides easy
+installation of other tools, including molecular dynamics tools such as
+OpenMM and ambertools, which are needed by ``openfe``. 
+Conda can be installed either with the `full Anaconda distribution
+<https://www.anaconda.com/products/individual>`_, or with
+the `smaller-footprint miniconda
+<https://docs.conda.io/en/latest/miniconda.html>`_. We also recommend
+`mamba
+<https://mamba.readthedocs.io/en/latest/installation.html#installation>`_ as
+a more perfomant variant of ``conda``.
+
+You can get ``openfe`` from the ``conda-forge`` channel. To install the most
+recent release, use the command ::
+
+  $ conda install -c conda-forge openfe
+
+With that, you should be ready to use ``openfe``!
```

Comment:
I'd sooner make them make an environment (best practices etc), so `conda create -n openfe -c conda-forge openfe` then `conda activate openfe`.  A bit slower, but avoids the hell of a base env with everything in it

Reply:
take a look at the update; I think this'll work